### PR TITLE
docs: Reorganise the main menu and the data menu

### PR DESCRIPTION
**Pull request by flavens**
resolves DEV-1258

### CODE REVIEW

**Comment by flavens** [comment]
@mdelez I cannot display the left-side menu as a drop-down menu like the one in DSP-API. Any idea about the syntax I should use?

File: mkdocs.yml [hunk 1]
```diff
@@ -35,7 +38,6 @@ theme:
     accent: 'deep purple'
   features:
     - navigation.tabs
-    - navigation.sections
```
Comment (flavens) [inline mkdocs.yml:38]: To display the left side menu with a drop-down, I removed this line. It is the only way I found to do what I wanted.
Open to any other suggestions.

**Comment by flavens** [comment]
@mdelez I resolved the conflicts. Possible to review when you can.

File: docs/user-guide/data/search.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# Search and browse
+
+DSP-APP offers the possibility for the user to search in 3 different ways: **full-text** search, **advanced** search, and **expert** search (Gravsearch query). The search bar is always available in the header of each page, whether logged in or out.
+
+![Search bar with the 3 search modes: full-text, advanced, and expert](../../assets/images/search-3modes.png)
+
+## Full-text search
+
+The full-text search performs queries including one or more terms or phrases and returns data that matches search conditions. By default, the search is performed in all projects stored in DSP. However, it is possible to filter by the project using the menu "Filter by project" on the left side of the search bar.
```
Comment (mdelez) [inline docs/user-guide/data/search.md:9]: The full-text search performs queries including one or more terms or phrases and returns data that matches **the** search conditions. By default, the search is performed in all projects stored in DSP. However, it is possible to filter by ~~the~~ project using the ~~menu~~ "Filter by project" **menu** on the left side of the search bar.

File: docs/user-guide/data/search.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+![Filter your full-text search by specific project](../../assets/images/search-fulltext-filterByProject.png)*<https://admin.dasch.swiss> - Search 1: Full-text search*
+
+When clicking on the search bar, the search history panel is displayed. The **10** last searches are registered. It is also possible to clear the search history list (*Clear list* button at the bottom of the panel or the *x* at the end of each line).
```
Comment (mdelez) [inline docs/user-guide/data/search.md:13]: the 10 last -> the last 10

File: docs/user-guide/data/work-on-data.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+
+If you don't have the permissions you were supposed to get, please contact the [DaSCH team](mailto:support@dasch.swiss).
+
+![Get the resource permissions.](../../assets/images/resource-permissions.png)*Check the resource permissions you are granted.*
+
+## Display data
+
+### Display a resource
+
+DSP-APP offers different resource views for different media types (images, videos, audio, archives, text, and PDF files). You can access the resources from the list of search results. Depending on the media type, DSP-APP offers different tools to work on the resource.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:17]: PDF files -> document files

File: docs/user-guide/data/work-on-data.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+### Add a property value
+
+You can create a new property value **if your data model allows it**. The value should have a cardinality of 0-1, 0-N or, 1-N.
+A "plus" button is displayed to the right of the label or after the first value field. You fill in the form field and save the changes by clicking on the floppy disk button. You can undo (back arrow button) or cancel the changes (x button).
+
+![Add a new property value.](../../assets/images/add-value.png)*Add a new property value.*
+
+![Save or undo the changes.](../../assets/images/add-new-text-value.png)*Save or undo the changes you made.*
+
+If your property is linked to another resource, you can search for an existing value by typing the first letter of the resource label you are looking for. You can also create a new resource instance by clicking on the first option in the drop-down menu, `Create new: xxx`.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:120]: the first **three** letters

File: docs/user-guide/data/work-on-data.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+
+---
+
+## Edit your data
+
+&#9888; You must have the granted permission to proceed.
+
+### Edit a resource
+
+To edit the resource label, you reach the edit form by opening the resource menu on the right-side of the toolbar and clicking on `Edit label`.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:146]: To edit the resource label, ~~you reach the edit form by opening~~ **open** the resource menu on the right-side of the toolbar and ~~clicking~~ **click** on `Edit label`.

File: docs/user-guide/data/work-on-data.md [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+
+![Archive viewer](../../assets/images/zip-file.png)
+
+### Display resource properties
+
+By default, only important properties are shown in the resource viewer. To display them all, click on the "unfold" button in the resource toolbar.
+
+![Show or hide properties.](../../assets/images/show-hide-properties.png)*Show or hide properties from the toolbar.*
+
+To get more information about the property value, i.e. date of creation and author name, you mouse over the value and then on the info button.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:79]: ~~you mouse~~ **hover** over the value and then ~~on~~ **over** the info button.

File: docs/user-guide/data/work-on-data.md [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+
+![Get more information about the value](../../assets/images/value-info.png)*Get the creation date and the author name of the value.*
+
+### Display a value comment
+
+To display the existing value comment, you mouse over the property value and click on the comment button (3rd icon from the left).
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:85]: you mouse -> hover

File: docs/user-guide/data/work-on-data.md [hunk 9]
```diff
@@ -0,0 +1,242 @@
...
+
+![Mouse over the property value to see if there is a comment.](../../assets/images/comment-value.png)*Mouse over the property value to see if there is a comment and display it.*
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:87]: Mouse -> Hover

File: docs/user-guide/data/work-on-data.md [hunk 10]
```diff
@@ -0,0 +1,242 @@
...
+
+![Save or undo the changes.](../../assets/images/add-new-text-value.png)*Save or undo the changes you made.*
+
+If your property is linked to another resource, you can search for an existing value by typing the first letter of the resource label you are looking for. You can also create a new resource instance by clicking on the first option in the drop-down menu, `Create new: xxx`.
+
+![Add a new linked resource value.](../../assets/images/add-new-resource-value.png)
+
+### Add an image region
+
+To create a new region on an image, you click on the button Draw a region in the toolbar, then select the region you want on the image.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:126]: Draw a region -> "Draw a region"

File: docs/user-guide/data/work-on-data.md [hunk 11]
```diff
@@ -0,0 +1,242 @@
...
+
+### Edit a resource
+
+To edit the resource label, you reach the edit form by opening the resource menu on the right-side of the toolbar and clicking on `Edit label`.
+
+![Add a new linked resource value.](../../assets/images/edit-resource-label.png)
+
+### Edit a property value
+
+You can edit a property value from the resource viewer. You mouse over the value and click on the edit button.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:152]: You mouse -> Hover

File: docs/user-guide/data/work-on-data.md [hunk 12]
```diff
@@ -0,0 +1,242 @@
...
+
+&#9888; You must have the granted permission to proceed.
+
+### Delete or erase a resource
+
+To delete a resource, there are 2 possibilities:
+
+1. **Delete** resource: the resource is not searchable or findable in DSP-APP anymore but the data still exists in the database.
+
+2. **Erase** resource: the resource is **definitely** deleted from the database, it will be impossible to get it back.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:174]: definitely -> permanently

File: docs/user-guide/data/work-on-data.md [hunk 13]
```diff
@@ -0,0 +1,242 @@
...
+
+![Delete or erase a resource.](../../assets/images/delete-erase-resource.png)*Delete or erase a resource.*
+
+### Delete a property value
+
+You can delete a property value. You mouse over the value and click on the delete button. You have to confirm your choice. It is possible to comment on the deletion of the value in the confirmation pop-up window to explain why it is being deleted.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:180]: You mouse -> Hover

File: docs/user-guide/data/work-on-data.md [hunk 14]
```diff
@@ -0,0 +1,242 @@
...
+
+2. Link object:  it allows you to create an annotation (the description of the link object) that can point to one or more resources. You will prefer this option if you want the link to exist and describe it.
+
+### Internal linkage
+
+To link 2 resources (A is an image and B is a document in the example), you go to your resource A that will be linked, open the **Share** menu and click on the button `Copy internal link on clipboard`. You have copied the internal link of your resource A that will be used to make the link with your resource B.
+
+![Copy the internal link](../../assets/images/link_copy_internal_link.png)*Copy the internal link of the resource A.*
+
+Go to your resource B. You click on the **edit** button of the property value where you want to add the link (by mousing over the value content). This value must be of type **rich text**.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:206]: mousing -> hovering

File: docs/user-guide/data/work-on-data.md [hunk 15]
```diff
@@ -0,0 +1,242 @@
...
+
+![Link object resource](../../assets/images/link-object-resource-view.png)
+
+It is possible to add other resources later on by clicking on the `+` button in the `has Link to` property section.
+
+---
+
+## Share your data
+
+Each resource of your dataset gets an ARK URL (Archival Resource Key), a persistent identifier that will allow citing permanently your resource in papers, conference presentations, books, etc.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:238]: citing permanently -> you to permanently cite

**Review by mdelez — CHANGES_REQUESTED** [review]

File: docs/user-guide/data/work-on-data.md [hunk 16]
```diff
@@ -117,13 +117,13 @@ A "plus" button is displayed to the right of the label or after the first value
 
 ![Save or undo the changes.](../../assets/images/add-new-text-value.png)*Save or undo the changes you made.*
 
-If your property is linked to another resource, you can search for an existing value by typing the first letter of the resource label you are looking for. You can also create a new resource instance by clicking on the first option in the drop-down menu, `Create new: xxx`.
+If your property is linked to another resource, you can search for an existing value by typing the first three letter of the resource label you are looking for. You can also create a new resource instance by clicking on the first option in the drop-down menu, `Create new: xxx`.
```
Comment (mdelez) [inline docs/user-guide/data/work-on-data.md:120]: letter -> letters

**Review by mdelez — CHANGES_REQUESTED** [review]
just one small fix and then I will approve

**Review by mdelez — APPROVED** [review]

**Pull request merged**